ScanbotSDKExamples/NET/ClassicComponent.Droid/MainActivity.cs

```diff
@@ -0,0 +1,602 @@
+﻿using Android.Graphics;
+using Android.Content;
+using Java.Util;
+using AndroidNetUri = Android.Net.Uri;
+using Android.Util;
+using IO.Scanbot.Sdk.Barcode.Entity;
+using ClassicComponent.Droid.Activities;
+using ClassicComponent.Droid.Utils;
+
+using AndroidX.Core.Content;
+using IO.Scanbot.Sdk.UI.View.Genericdocument;
+using IO.Scanbot.Sdk.UI.Result;
+using IO.Scanbot.Sdk.UI.View.Genericdocument.Configuration;
+using IO.Scanbot.Genericdocument.Entity;
+using IO.Scanbot.Sdk.UI.View.Base;
+using ClassicalComponentsDemo.Droid;
+using DocumentSDK.MAUI.Constants;
+using SBSDK = DocumentSDK.MAUI.Native.Droid.ScanbotSDK;
+using DocumentSDK.MAUI.Models;
+
+namespace ClassicComponent.Droid
+{
+    [Activity(Label = "Scanbot SDK Example Xamarin", MainLauncher = true, Icon = "@mipmap/icon",
```

Comment:
@scan-mayank, in keeping with the name updates, I think we may want to remove Xamarin and also make the name consistent with all the other label changes for each app.